app/views/equipment_listings/show.html.erb

```diff
@@ -1 +1,14 @@
-<%= @equipment.name  %>
+<% if  @equipment_listing.nil? == false %>
+  <ul>
+    <li> equipment name        : <%= @equipment_listing.name %>(<%= @equipment_listing.id %>)
+    <li> equipment description : <%= @equipment_listing.description %>
+    <li> equipment category    : <%= @equipment_listing.category %>
+    <li> equipment hourly_rate : <%= @equipment_listing.hourly_rate %>
+    <li> equipment available   : <%= @equipment_listing.available %>
+    <li> equipment user_id     : <%= @equipment_listing.user_id %>
+  </ul>
+  <%= link_to "Back to Index", equipment_listings_path %> |
+  <%= link_to "Book this item", new_equipment_listing_booking_path(@equipment_listing) %>
+<% else %>
+  No item found for ID <% @equipment_listing_id %>
```

Comment:
Please change  <% @equipment_listing_id %> to  <%= @equipment_listing.id %>

Reply:
thank you for checking.
However if I do `<%= @equipment_listing.id %>` I get the below error.
I tried `<%= @equipment_listing_id %>` , and it worked as expected.


![image](https://user-images.githubusercontent.com/37240055/180215646-9a05e895-171b-4904-97c2-fae41880b16b.png)

![image](https://user-images.githubusercontent.com/37240055/180216088-680d0057-a190-411a-b58c-60847d3fb233.png)